### PR TITLE
Update sodiumoxide to v0.2

### PR DESCRIPTION
**Pull request by kpp**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,7 +49,6 @@ matrix:
 
     - os: linux
       rust: stable
-      sudo: required
```
Comment (kpp) [inline .travis.yml]: `sudo: required` for kcov. whereas for the root conf and for all other runners you should set `sudo: false`.

File: README.md [hunk 2]
```diff
@@ -28,42 +28,42 @@ issue / pull request should be filled on the reference repository.
...
 [CONTRIBUTING.md](/CONTRIBUTING.md).
 
-## Dependencies
-| **Name** | **Version** |
-|----------|-------------|
-| libsodium | >=1.0.13 |
-
 ## Building
-Fairly simple. You'll need [Rust] >= 1.26.0 and [libsodium].
+Fairly simple. First, install [Rust] >= 1.30.1.
```
Comment (kurnevsky) [inline README.md]: Do we also need clang now?

**Comment by kurnevsky** [comment]
`increment_nonce_number` can use native libsodium function now.

File: .travis.yml [hunk 3]
```diff
@@ -41,6 +36,7 @@ matrix:
   include:
     - os: linux
       rust: nightly
+      sudo: false
```
Comment (kurnevsky) [inline .travis.yml]: Why disable it? And does it have to be disabled twice?

File: README.md (hunk 2 above)
Comment (kpp) [reply README.md]: No.

Comment (kurnevsky) [reply README.md]: ok, but gcc should be required, shouldn't it? https://github.com/alexcrichton/cc-rs#compile-time-requirements

Comment (suhr) [reply README.md]: Rust actually asks you to install Build Tools for Visual Studio on Windows.

Comment (kurnevsky) [reply README.md]: Don't you think it should be mentioned here?

**Pull request merged**